### PR TITLE
docs: add meeting minutes for 2019-10-22

### PR DESCRIPTION
**Pull request by ZauberNerd**

### CODE REVIEW

File: 2019-10-22.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+- Create list of next meetup dates
+- Plan the next meetup
+
+## Participants
+
+...
+
+## Minutes
+
+### Reflection of the last meetup (Finanzcheck)
```
Comment (herschel666) [inline 2019-10-22.md]: The last meetup has been at Wunder.

**Review by learosema — DISMISSED** [review]

**Comment by ZauberNerd** [comment]
Meeting minutes have been added. Please review again @hamburg-js/organizers

**Review by agalliat — DISMISSED** [review]

File: 2019-10-22.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+## Minutes
+
+### Reflection of the last meetup (Wunder)
+
+- __(i)__ We had only one proposal available for that day, thankfully Martin spontaneously jumped in to give a second talk
+- __(d)__ We will try to increase our communication efforts through twitter & slack and post more regularly call for papers
+- __(i)__ The talks and location (Wunder) were great and feedback was positive
+- __(i)__ Sometimes Q&A sessions drag out between a few participants and the speaker by sliding into a back and forth questioning
+- __(d)__ We will try to be more aware of than, when that happens and redirect it to the break after the Q&A has ended
```
Comment (ZauberNerd) [inline 2019-10-22.md]:
```suggestion
- __(d)__ We will try to be more aware of these situations and when they happen redirect them to conversions during the breaks
```

**Review by agalliat — DISMISSED** [review]

**Review by herschel666 — CHANGES_REQUESTED** [review]
found one typo.

File: 2019-10-22.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+## Minutes
+
+### Reflection of the last meetup (Wunder)
+
+- __(i)__ We had only one proposal available for that day, thankfully Martin spontaneously jumped in to give a second talk
+- __(d)__ We will try to increase our communication efforts through twitter & slack and post more regularly call for papers
+- __(i)__ The talks and location (Wunder) were great and feedback was positive
+- __(i)__ Sometimes Q&A sessions drag out between a few participants and the speaker by sliding into a back and forth questioning
+- __(d)__ We will try to be more aware of these situations and when they happen redirect them to conversions during the breaks
```
Comment (herschel666) [inline 2019-10-22.md]:
```suggestion
- __(d)__ We will try to be more aware of these situations and when they happen redirect them to conversations during the breaks
```

**Review by herschel666 — APPROVED** [review]

**Pull request merged**